angular-rails-app/app/assets/javascripts/property/property.html

```diff
@@ -1,8 +1,7 @@
 <section style="min-height: 65px;margin-top:105px;">
   <div class="container">
     <p id="breadcrumb">
-      <a class="breadcrumb_top" ui-sref="home">Home /</a> Building /
-      {{vm.data.street_address | titleCase}}, {{vm.data.city}} {{vm.data.state}}
+      <a class="breadcrumb_top" ui-sref="home">Home /</a> Building 
```

Comment:
Yes - it was originally.  However, at our last meeting Andrew told me to get rid of it.  So, I actually made this change with him in person.